dashboard/app/controllers/levels_controller.rb

```diff
@@ -548,6 +548,9 @@ def extra_links
           if @level.class != Music
             links[@level.name] << {text: "e[x]emplar", url: edit_exemplar_level_path(@level), access_key: 'x'}
           end
+          if [Music].include?(@level.class)
```

Comment:
Maybe out of scope, but wondering if we should have functions like `supports_toolbox_mode?`, `supports_start_mode?` etc on the individual models, rather than having to manage lists here?

Reply:
Yeah, that's fair. We already have a lot of similar list checking within this function for start mode (`[Javalab, Music, Pythonlab, Weblab2]`) and exemplar mode (ditto but `!= Music`). Adding functions for each of these on the models seems like a good refactor though!